src/main/java/org/alfresco/repo/search/impl/querymodel/impl/db/DBResultSet.java

```diff
@@ -62,7 +62,9 @@
     
     private SimpleResultSetMetaData resultSetMetaData;
     
-    private BitSet prefetch;
+    private BitSet prefetch;
+
+	private int numberFound;

```

Comment:
Use spaces instead of tab at start of the line.

Reply:
Also this is due to formatting issues I had as mentioned above. It's fixed now.